contrib/neovide/update.py

```diff
@@ -0,0 +1,2 @@
+url = "https://github.com/neovide/neovide/tags"
```

Comment:
this should probably work automatically

Reply:
nah the rest of the URLs clobber it for some reason.

will try again tho.

Reply:
Nah it doesn't seem to be working

without `update.py` -
```
$ cbuild update-check contrib/neovide verbose
Checking for updates: neovide=0.12.1
Adding 'https://neovide.dev' for version check...
Adding 'https://github.com/neovide/neovide/archive/refs/tags/' for version check...
Adding 'https://gn.googlesource.com/gn/+archive/' for version check...
Adding 'https://github.com/rust-skia/skia/archive/' for version check...
Adding 'https://github.com/google/wuffs-mirror-release-c/archive/' for version check...
Adding 'https://github.com/KhronosGroup/SPIRV-Cross/archive/' for version check...
Adding 'https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator/archive/' for version check...
Fetching 'https://neovide.dev' for version checks...
Fetching 'https://github.com/neovide/neovide/tags' for version checks...
Fetching 'https://gn.googlesource.com/gn/+archive/' for version checks...
Fetching 'https://github.com/rust-skia/skia/tags' for version checks...
Fetching 'https://github.com/google/wuffs-mirror-release-c/tags' for version checks...
Fetching 'https://github.com/KhronosGroup/SPIRV-Cross/tags' for version checks...
Fetching 'https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator/tags' for version checks...
Checking found version: 0.3.0
Checking found version: 0.3.1
Checking found version: 0.3.2
Checking found version: 0.3.3
Checking found version: 0.10.0
Checking found version: 0.10.1
Checking found version: 0.10.2
Checking found version: 0.10.3
Checking found version: 0.10.4
Checking found version: 0.11.0
Checking found version: 0.11.1
Checking found version: 0.11.2
Checking found version: 0.12.0
Checking found version: 0.12.1
Checking found version: 1.0.0
Checking found version: 1.0.1
Checking found version: 2.0.0
Checking found version: 2.1.0
Checking found version: 2.2.0
Checking found version: 2.3.0
Checking found version: 3.0.0
Checking found version: 3.0.1
contrib/neovide: 0.12.1 -> 1.0.0
contrib/neovide: 0.12.1 -> 1.0.1
contrib/neovide: 0.12.1 -> 2.0.0
contrib/neovide: 0.12.1 -> 2.1.0
contrib/neovide: 0.12.1 -> 2.2.0
contrib/neovide: 0.12.1 -> 2.3.0
contrib/neovide: 0.12.1 -> 3.0.0
contrib/neovide: 0.12.1 -> 3.0.1
```